standalone/include/psen_scan_v2_standalone/scanner_config_builder.h

```diff
@@ -67,8 +67,8 @@ inline ScannerConfiguration ScannerConfigurationBuilder::build() const
 
 uint32_t ScannerConfigurationBuilder::convertIP(const std::string& ip)
 {
-  const auto ip_number = inet_network(ip.c_str());
-  if (static_cast<in_addr_t>(-1) == ip_number)
+  const auto ip_number = inet_addr(ip.c_str());
```

Comment:
On windows I have to revert the IP_ADDRESS of both host and sensor ip in order to get this working.
(e.g. "192.168.0.100" -> "100.0.168.192").

Reply:
or drop the `_byteswap_*` in https://github.com/PilzDE/psen_scan_v2/pull/141/commits/67049a872e9ca20a45cc427700fb86d423f133fe# ?

Reply:
See 64da1d9 - @agutenkunst could you please test, if that sends the correct ip address on both linux and windows?

Reply:
Concerning the whole host byte order topic I suggests postponing until this is in upstream. If we get this tests up and running  with Windows these refactorings can be easily integrated.